routes/api/utils/utils.js

```diff
@@ -4,6 +4,7 @@ async function handleUpDelRes(resPromise, res) {
 }
 
 function checkDupErr(err, res) {
+    console.log(err);
```

Comment:
this is fine, but just so you know you can also use `console.info` and `console.warn` and `console.error` when logging or better yet a nice logging library instead of using console